support-frontend/assets/pages/supporter-plus-landing/components/paymentTsAndCs.tsx

```diff
@@ -157,6 +175,36 @@ export function PaymentTsAndCs({
 		</div>
 	);
 	const productLabel = productCatalogDescription[productKey].label;
+	const subscriptionBasis = !isStudentOneYearRatePlan
+		? ' on a subscription basis'
+		: '';
+	const supporterPlusStartTsAndCs = `If you pay at least ${legalPrice}, you will receive the ${productLabel} benefits${subscriptionBasis}. If you increase your payments per ${billingPeriodSingular}, these additional amounts will be separate ${billingPeriodPlural} voluntary financial contributions to the Guardian. `;
```

Comment:
I think this copy is shown for both students and non-students. Is the bit about increasing your payment relevant for the one year student offer?